ATC/src/Radar.cpp

```diff
@@ -2,6 +2,30 @@
 
 #include <sys/siginfo.h>
 
+Radar::Radar(std::vector<Plane> &planes)
+: planes(planes){}
+
+PlanePositionResponse Radar::pingPlane(int planeNumber, PlanePositionResponse *out) {
+	for (size_t i = 0; i < planes.size(); i++) {
+		if (planes[i].getPlaneId() == planeNumber) {
+			*out = pingPlane(planes[i]);

```

Comment:
what's the purpose of overloading the function here?

Reply:
I kept the original since both this and `pingAirspace` call the `pingPlane` overload that takes a `Plane` object.